knowledge-base/how-to-split-a-nested-tabel-on-two-pages.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How to split a nested table on two pages
+description: A common requirement when using nested tables in the report is to force them to break between pages. 
+type: how-to
+page_title: Render nested tables on multiple pages
+slug: how-to-split-a-nested-tabel-on-two-pages
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>All</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
```

Comment:
You can add the report element here as well. E.g. Report Element - Nested Table/Table